Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java

```diff
@@ -47,10 +47,10 @@ public void onResume(@NonNull final Activity activity) {
             super.onResume(activity);
         }
     };
-    private static VungleRouter sInstance = new VungleRouter();
+    private static final VungleRouter sInstance = new VungleRouter();
     private static SDKInitState sInitState = SDKInitState.NOTINITIALIZED;
-    private static Map<String, VungleRouterListener> sVungleRouterListeners = new HashMap<>();
-    private static Map<String, VungleRouterListener> sWaitingList = new HashMap<>();
+    private static final Map<String, VungleRouterListener> sVungleRouterListeners = new HashMap<>();
+    private static final Map<String, VungleRouterListener> sWaitingList = new HashMap<>();
```

Comment:
one more thing - if we're making these `final`, they should be renamed, - INSTANCE, WAITING_LIST, VUNGLE_ROUTER_LISTENERS